.github/workflows/test.yaml

```diff
@@ -17,7 +17,12 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-20.04
+    name: Build and test on OS - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-latest", "macos-latest" ]
```

Comment:
cool! this will require adding an input to https://github.com/uwhackweek/jupyterbook-template/blob/main/.github/actions/setupconda/action.yml to specify using either `conda-linux-64.lock` or `conda-osx-64.lock`

Reply:
Working on it right now ...